<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QTableWidget" name="mon_tableWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>821</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 198, 198);
color: rgb(0, 72, 0);</string>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="rowCount">
    <number>15</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>NO.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">Filesystem</string>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Used</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Available</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Usage %</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mounted on</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignRight|AlignVCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="start_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>95</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>95</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 206, 255);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>95</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 206, 255);</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 30pt &quot;FreeSans&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>DISK MONITOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="mail_stat_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(123, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>stop_pushButton</zorder>
  <zorder>mon_tableWidget</zorder>
  <zorder>clear_pushButton</zorder>
  <zorder>exit_pushButton</zorder>
  <zorder>label</zorder>
  <zorder>start_pushButton</zorder>
  <zorder>mail_stat_label</zorder>
 </widget>
 <tabstops>
  <tabstop>mon_tableWidget</tabstop>
  <tabstop>stop_pushButton</tabstop>
  <tabstop>start_pushButton</tabstop>
  <tabstop>clear_pushButton</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clear_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mon_tableWidget</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>start_pushButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>stop_pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>start_pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>stop_pushButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mon_tableWidget</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
